package.json

```diff
@@ -16,7 +16,7 @@
     "lint:typescript": "eslint -c types/.eslintrc.json types/**/*.d.ts test/types/**/*.test-d.ts",
     "prepublishOnly": "tap --no-check-coverage test/internals/version.test.js",
     "test": "npm run lint && npm run unit && npm run test:typescript",
-    "test:ci": "npm run unit -- --cov --coverage-report=lcovonly && npm run test:typescript",
+    "test:ci": "npm run unit -- -R terse --cov --coverage-report=lcovonly && npm run test:typescript",
```

Comment:
I'll call this out before anyone else:

I added this because one of the macOS runs flaked. We have so many tests that scrolling through them to figure out _why_ that run flaked was effectively impossible. By using `-R terse`, only the failures will be published to the test run logs. This will make it far easier to find the problems.

Reply:
THANKS!!! I didn't know this was possiblez